vllm/transformers_utils/tokenizer.py

```diff
@@ -47,13 +51,14 @@ def encode_tokens(
     Backend-agnostic equivalent of HF's
     :code:`tokenizer.encode(text, add_special_tokens=...)`.
     """
-    if isinstance(tokenizer, MistralTokenizer):
-        return tokenizer.tokenizer.encode(text,
-                                          bos=add_special_tokens,
-                                          eos=add_special_tokens)
-    elif add_special_tokens is not None:
+    if isinstance(tokenizer, PreTrainedTokenizerBase):
```

Comment:
switched the `if-else` so that we don't need to import `MistralTokenizer` too early.